CONTRIBUTING.md

```diff
@@ -230,14 +230,14 @@ You can test Your Installation with the following steps.  These steps assume you
 * Create a branch of the FIBO repository (in Sourcetree, click the "branch" icon).  Name your branch SerializerTest.
 * In a text editor, open the file fibo/BE/LegalEntities/LegalPersons.rdf
 * Find the definition of LegalPerson in this file; it looks like this: 
-```<owl:Class rdf:about="&fibo-be-le-lp;LegalPerson">```
+```<owl:Class rdf:about="&cmns-org;LegalPerson">```
```

Comment:
not a good example since this class is in an entirely different file in a different repo, not the one just opened in the previous instruction. Choose a class actually in the file